src/main/scala-2.12/org/apache/james/gatling/smtp/SmtpProtocol.scala

```diff
@@ -24,12 +26,13 @@ object SmtpProtocol {
     override def onExit: Session => Unit = s => ()
   }
 
-  val DEFAULT_PORT = 25
-  val DEFAULT_PORT_SSL = 465
+  val PORT = Properties.envOrElse("SMTP_PORT", "25").toInt
```

Comment:
587 to be able to test using authentication ?

Reply:
I'm only focusing on parametrizing it, not changing underlying behavior